def write_error(error):
    with open('errors.txt', 'a') as f:
        f.write(error)
